lib/condor.py

```diff
@@ -20,19 +20,69 @@
 import re
 import random
 import subprocess
+from contextlib import contextmanager
 from typing import Dict, List, Any, Tuple, Optional, Union
 
 import htcondor  # type: ignore
 import classad  # type: ignore
 
 import packages
+import fake_ifdh
 
 random.seed()
 
 # pylint: disable-next=no-member
 COLLECTOR_HOST = htcondor.param.get("COLLECTOR_HOST", None)
 
 
+@contextmanager  # type: ignore
+def submit_vt(vo: str, role: str, schedd: str, verbose: int) -> None:
+
+    """
+    Rearrange vaulttoken files around a submit
+    Rename them to keep timestamps, etc.
+    """
+    try:
+        tmp = os.environ.get("TMPDIR", "/tmp")
+        uid = os.getuid()
+        pid = os.getpid()
+        if verbose > 1:
+            print("pre-pre:")
```

Comment:
Everything looks good except for these "pre-pre", "post-pre", etc. statements, IMO.  Could you please change that to something more descriptive so that users (and future us) know what it's referring to if they look at the debug output?

Reply:
Well, I knew what ti meant ;-) . Good call.